### PR TITLE
Allow the publication name to be altered from the extension

### PR DESCRIPTION
**Pull request by StylianosGakis**
Context:
#3

### CODE REVIEW

File: src/main/kotlin/nmcp/NmcpPublishTask.kt [hunk 1]
```diff
@@ -48,7 +52,7 @@ abstract class NmcpPublishTask : DefaultTask() {
         val body = MultipartBody.Builder()
             .addFormDataPart(
                 "bundle",
-                "publication.zip",
+                "${publicationName.orElse("publication")}.zip",
```
Comment (martinbonnin) [inline src/main/kotlin/nmcp/NmcpPublishTask.kt:55]: Should we default to `${project.name}-${project.version}.zip` instead?

Comment (StylianosGakis) [reply src/main/kotlin/nmcp/NmcpPublishTask.kt]: Honestly that is even better yes. Just pushed it

File: src/main/kotlin/nmcp/NmcpPublishTask.kt [hunk 2]
```diff
@@ -48,7 +51,7 @@ abstract class NmcpPublishTask : DefaultTask() {
         val body = MultipartBody.Builder()
             .addFormDataPart(
                 "bundle",
-                "publication.zip",
+                "$publicationName.zip",
```
Comment (martinbonnin) [inline src/main/kotlin/nmcp/NmcpPublishTask.kt:54]:
```suggestion
                publicationName.get(),
```

**Comment by StylianosGakis** [comment]
Published locally with the current state of the project, after the commit 9a67540
And I got this result. So this seems to work very well now.
image

I tried to fix the last comment you had in the PR as well. Let me know if I didn't do it as it should be.

**Comment by martinbonnin** [comment]
Sweet ! Thanks for the contribution ❤️ , I'll do a release in a bit.

**Comment by martinbonnin** [comment]
Version is building [here](https://github.com/GradleUp/nmcp/actions/runs/8493609768/job/23267872452)

**Pull request merged**